Emby.Naming/Audio/AlbumParser.cs

```diff
@@ -22,7 +23,7 @@ public bool IsMultiPart(string path)
         {
             var filename = Path.GetFileName(path);
 
-            if (string.IsNullOrEmpty(filename))
+            if (path.Length == 0)
```

Comment:
This could move to before the call to `Path.GetFileName()` now

Reply:
Path.GetFileName will return and empty string if the last character of path is a directory or volume separator character

Reply:
Did you mean to use `filename` here then instead of `path`?

Reply:
Whoops, yes